django-transparent/entry.sh

```diff
@@ -6,14 +6,16 @@ cat > $DJANGOPROJECT_DATA_DIR/conf/secrets.json <<EOF
 {
   "secret_key": "$(dd if=/dev/urandom bs=4 count=16 2>/dev/null | base64 | head -c 32)",
   "superfeedr_creds": ["email@example.com", "some_string"],
-  "db_host": "$POSTGRES_HOST",
+  "db_host": "$ACRA_SERVER_HOST",
+  "db_port": "$ACRA_SERVER_PORT",
   "db_password": "$POSTGRES_DJANGO_PASSWORD",
-  "trac_db_host": "$POSTGRES_HOST",
-  "trac_db_password": "$POSTGRES_DJANGO_PASSWORD"
+  "trac_db_host": "$ACRA_SERVER_HOST",
+  "trac_db_password": "$POSTGRES_DJANGO_PASSWORD",
+  "trac_db_port": "$ACRA_SERVER_PORT"
 }
 EOF
 
-export PGSSLMODE=disable
+export PGSSLMODE=require
```

Comment:
why haven't used `require-full`?

Reply:
You mean `verify-full`?

 Do we actually need to set `verify-full` mode here as we do it just for creating some schema directly inside DB, I think we can even return it back to `disable` mode. What do you think @Lagovas?

Reply:
it's good example with secure scheme of communication) especially if it easy to show.
and not, you will take errors if you turn on `disable` )) because of client require TLS and db will not answer that it doesn't support it, and connection will be terminated.

Reply:
Got it, let me add `verify-full` mode here, than:)